src/ui/public/kbn_top_nav/kbn_top_nav.js

```diff
@@ -50,7 +50,7 @@ module.directive('kbnTopNav', function (Private) {
       // It will no accept any programatic way of setting its name
       // besides this because it happens so early in the digest cycle
       return `
-        <navbar class="kibana-nav-options">
+        <navbar ng-show="chrome.getVisible()" class="kibana-nav-options">
```

Comment:
Ideally yes, but if we ng-if then the timepicker isn't instantiated and we won't get things like auto refresh.